### PR TITLE
Signal errors

### PR DESCRIPTION
**Pull request by hseeberger**

### CODE REVIEW

**Comment by hseeberger** [comment]
Verified, see attached picture.

screen shot 2015-05-06 at 12 19 59

**Comment by hseeberger** [comment]
@huntc, up for review/merge.
@2m, please take this as guideline for the respective changes in the CLI.

**Comment by huntc** [comment]
Nice! How about colouring the line red?

File: src/main/scala/com/typesafe/conductr/sbt/console/Screen.scala [hunk 1]
```diff
@@ -86,7 +86,13 @@ class Screen(refresh: Boolean) extends Actor with ImplicitFlowMaterializer {
     println(allColumns.map(_.titleForPrint).reduce(_ + _).invert.render)
 
     val rowCounts = allColumns.map(_.data.map(_.size)).transpose.map(_.max)
-    val lines = allColumns.map(_.dataForPrint(rowCounts)).transpose
+    val lines = {
+      val lines = allColumns.map(_.dataForPrint(rowCounts)).transpose
+      if (bundles.exists(_.hasError))
+        lines :+ Vector("There are errors: use `conduct events` or `conduct logs` for further information")
```
Comment (huntc) [inline src/main/scala/com/typesafe/conductr/sbt/console/Screen.scala]: Nice

Comment (huntc) [inline src/main/scala/com/typesafe/conductr/sbt/console/Screen.scala]: We should add conduct `events` and `logs` sub-commands and make them yield a message about them coming along soon. WDYT?

Comment (hseeberger) [inline src/main/scala/com/typesafe/conductr/sbt/console/Screen.scala]: Martynas is already at it.

**Comment by hseeberger** [comment]
Red? Great idea! Or maybe pink? ;-)

**Comment by huntc** [comment]
I think [Ochre](http://en.wikipedia.org/wiki/Ochre) - a localised red - very Australian.

I'm joking. :-) Red is good.

**Comment by hseeberger** [comment]
OK, red it is (even the !):

screen shot 2015-05-06 at 12 35 31

**Comment by huntc** [comment]
Nice! Do you think the whole line, or at least the bundle id would also benefit from from being red?

**Comment by hseeberger** [comment]
Yeah, let's also color the bundle ID.

**Comment by hseeberger** [comment]
Done.

**Comment by 2m** [comment]
LGTM

**Comment by huntc** [comment]
Nice work Heiko!

**Pull request merged**